bindings/java/nz/mega/sdk/MegaApiJava.java

```diff
@@ -59,6 +59,7 @@ void runCallback(Runnable runnable) {
     static Set<DelegateMegaTransferListener> activeTransferListeners = Collections.synchronizedSet(new LinkedHashSet<DelegateMegaTransferListener>());
     static Set<DelegateMegaGlobalListener> activeGlobalListeners = Collections.synchronizedSet(new LinkedHashSet<DelegateMegaGlobalListener>());
     static Set<DelegateMegaListener> activeMegaListeners = Collections.synchronizedSet(new LinkedHashSet<DelegateMegaListener>());
+    static Set<DelegateMegaLogger> activeMegaLoggers = Collections.synchronizedSet(new LinkedHashSet<DelegateMegaLogger>());
```

Comment:
Since this will be used to keep the references to the listeners, the `logger` variable above seems to be useless now:
`static DelegateMegaLogger logger;`